event/consumer_test.go

```diff
@@ -6,37 +6,40 @@ import (
 	"github.com/ONSdigital/dp-hierarchy-builder/event"
 	"github.com/ONSdigital/dp-hierarchy-builder/event/eventtest"
 	"github.com/ONSdigital/dp-import/events"
+	kafka "github.com/ONSdigital/dp-kafka"
+	"github.com/ONSdigital/dp-kafka/kafkatest"
 	"github.com/ONSdigital/dp-reporter-client/reporter/reportertest"
-	"github.com/ONSdigital/go-ns/kafka"
-	"github.com/ONSdigital/go-ns/kafka/kafkatest"
-	"github.com/ONSdigital/go-ns/log"
+	"github.com/ONSdigital/log.go/log"
 	. "github.com/smartystreets/goconvey/convey"
 	"testing"
 	"time"
 )
 
+var ctx = context.Background()
```

Comment:
Due to being a global variable would it be better to name this variable to something like `testCtx` ?

Just concerned it may get picked up by app due to existing in same package.

Reply:
The tests have their own package in this case - I have changed the name still to make it clear it's for test purposes.

Reply:
Thanks, I think i tend to use the same package name for test files as well as the original file <- my own bad practice;